libs/src/api/serviceProvider.js

```diff
@@ -1,4 +1,4 @@
-const _ = require('lodash')
+const { sampleSize } = require('lodash')
```

Comment:
so does this load just the function from the required module, or is everything still loaded with the 'require' statement and we just use this one function?

Reply:
tl;dr it shouldn't really matter, but here's a good discussion
https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash

It depends on what bundler is being used to bundle this code. In the dapp, we're well past webpack2 and we have tree shaking, so it should iron all of it out for us. But since libs doesn't actually include a bundler (which it should), you can get weirdness here.